src/main/java/edu/ie3/datamodel/io/factory/deserializing/Deserializer.java

```diff
@@ -0,0 +1,56 @@
+package edu.ie3.datamodel.io.factory.deserializing;
+
+import edu.ie3.datamodel.models.OperationTime;
+import edu.ie3.datamodel.models.UniqueEntity;
+import org.locationtech.jts.geom.Point;
+
+import javax.measure.Quantity;
+import java.time.ZonedDateTime;
+import java.util.*;
+
+public class Deserializer {
+
+    // TODO pre-fill with mappings like Quantity:QuantityDeserializingStrategy
+    private static Map<Class, DeserializingStrategy> deserializingStrategies = new HashMap<>();
+    static {
+        deserializingStrategies.putAll(PrimitiveDeserializingStrategies.PRIMITIVE_STRATEGIES);
+        deserializingStrategies.put(Point.class, new PointDeserializingStrategy());
+        deserializingStrategies.put(Quantity.class, new QuantityDeserializingStrategy());
+        deserializingStrategies.put(UUID.class, new UuidDeserializingStrategy());
+        deserializingStrategies.put(ZonedDateTime.class, new ZonedDateTimeDeserializingStrategy());
+        deserializingStrategies.put(OperationTime.class, new OperationTimeDeserializingStrategy());
+    }
+
+
+    public static <T> Optional<T> deserialize(Class<T> cls, Map<String, String> fieldMap) {
+        return deserialize(cls, fieldMap, Collections.emptyMap());
+    }
+
+    public static <T> Optional<T> deserialize(Class<T> cls, Map<String, String> fieldMap, Map<String, Object> deserializedFields) {
+        DeserializingStrategy<T> deserializingStrategy = getDeserializingStrategy(cls);
+        if(deserializingStrategy == null) return Optional.empty();
```

Comment:
Wouldn't an Exception be better? Otherwise you might only see the missing deserialization strategy by having a detailed look at the content of the data, which might be errorprone?